client/python/samples/README.md

```diff
@@ -222,12 +237,215 @@ optional arguments:
   --model_version MODEL_VERSION Model version to query. default: latest available
   --output_save_path OUTPUT_SAVE_PATH Path to store output.
+  --timeout TIMEOUT Request timeout. default: 10.0
 ```
 
 - Usage example
 
 ```bash
-python vehicle_detection_predict_binary.py --images_dir ./images/cars/ --output_save_path ./output
+python vehicle_detection_predict_binary_grpc.py --images_dir ./images/cars/ --output_save_path ./output
+Making directory for output: ./output
+Detection results in file:  ./output/road1.jpg
+```
+
+## HTTP Client Examples <a name="http-api"></a>
+
+### Model Status <a name="model-status-http">
+
+#### **Get information about the status of served models over HTTP interace:**
+
+- Command
+
+```bash
+python get_http_model_status.py -h
+usage: get_http_model_status.py [-h] [--http_address HTTP_ADDRESS]
+                                [--http_port HTTP_PORT]
+                                [--model_name MODEL_NAME]
+                                [--model_version MODEL_VERSION]
+                                [--timeout TIMEOUT]
+
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --http_address HTTP_ADDRESS Specify url to http service. default:localhost
+  --http_port HTTP_PORT Specify port to http service. default: 9000
+  --model_name MODEL_NAME Model name to query. default: resnet
+  --model_version MODEL_VERSION Model version to query. Lists all versions if omitted
+  --timeout TIMEOUT Request timeout. default: 10.0
+```
+
+- Usage Example
+
+```
+python get_http_model_status.py --http_port 9000 --model_name resnet
+{1: {'state': 'AVAILABLE', 'error_code': 0, 'error_message': 'OK'}}
+```
+
+
+### Model Metadata <a name="model-metadata-http">
+
+#### **Get information about the status of served models over HTTP interace:**
+
+- Command
+
+```bash
+python get_http_model_metadata.py --help
+usage: get_http_model_metadata.py [-h] [--http_address HTTP_ADDRESS]
+                                  [--http_port HTTP_PORT]
+                                  [--model_name MODEL_NAME]
+                                  [--model_version MODEL_VERSION]
+                                  [--timeout TIMEOUT]
+
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --http_address HTTP_ADDRESS Specify url to http service. default:localhost
+  --http_port HTTP_PORT Specify port to http service. default: 9000
+  --model_name MODEL_NAME Model name to query. default: resnet
+  --model_version MODEL_VERSION Model version to query. Lists all versions if omitted
+  --timeout TIMEOUT Request timeout. default: 10.0
+```
+- Usage Example
+
+```
+python get_http_model_metadata.py --http_port 9000 --model_name resnet --model_version 1
+{1: {'inputs': {'map/TensorArrayStack/TensorArrayGatherV3': {'shape': [1, 224, 224, 3], 'dtype': 'DT_FLOAT'}}, 'outputs': {'softmax_tensor': {'shape': [1, 1001], 'dtype': 'DT_FLOAT'}}}}
```

Comment:
The response structure has been simplified and the result should look like this. Something is wrong either in the script or the implementation.